app/questionnaire/path_finder.py

```diff
@@ -180,6 +180,10 @@ def get_blocks(self):
             if repeating_rule:
                 no_of_repeats = evaluate_repeat(repeating_rule, self.answer_store)
 
+            if no_of_repeats > 50:
```

Comment:
Can we have this in a settings file please (the count on the number of repeats allowed)